resources/less/virkailija-common.less

```diff
@@ -23,3 +23,18 @@
 .close-details-button:hover > .close-details-button-mark {
   color: white;
 }
+
+.privilege-info-outer {
+  position: relative;
+  padding: 0 10px 0 10px;
+}
+
+.privilege-info-inner {
+  background-color: #def2ff;
+  text-align: center;
+  display: block;
+  color: #2a2a2a;
```

Comment:
Would be nice if (both of the) colors would variables (the other color is already defined in vars.less, though maybe named a bit un-reusably).